lib/Doctrine/ruleset.xml

```diff
@@ -205,18 +205,21 @@
     <rule ref="SlevomatCodingStandard.Commenting.RequireOneLinePropertyDocComment"/>
     <!-- Forbid assignments in conditions -->
     <rule ref="SlevomatCodingStandard.ControlStructures.AssignmentInCondition"/>
-    <!-- Require consistent spacing for control structures -->
-    <rule ref="SlevomatCodingStandard.ControlStructures.ControlStructureSpacing">
```

Comment:
This should likely stay:
```php
if (...) {
   // ...
}
echo 'foo';
```

Should become:

```php
if (...) {
   // ...
}

echo 'foo';
```

Reply:
@Ocramius this is handled by https://github.com/doctrine/coding-standard/pull/137

Reply:
Alright 👍